Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -1161,4 +1167,21 @@ void MolDraw2D::setupTextDrawer() {
   }
 }
 
+std::pair<int, int> MolDraw2D::getMolSize(
+    const ROMol &mol, const std::string &legend,
+    const std::vector<int> *highlight_atoms,
+    const std::vector<int> *highlight_bonds,
+    const std::map<int, DrawColour> *highlight_atom_map,
+    const std::map<int, DrawColour> *highlight_bond_map,
+    const std::map<int, double> *highlight_radii, int confId) {
+  setupTextDrawer();
+  MolDraw2D_detail::DrawMol dm(
+      mol, legend, panelWidth(), panelHeight(), drawOptions(), *text_drawer_,
+      highlight_atoms, highlight_bonds, highlight_atom_map, highlight_bond_map,
+      nullptr, highlight_radii, supportsAnnotations(), confId);
+  dm.setOffsets(x_offset_, y_offset_);
```

Comment:
This translates all the draw objects by the offsets, but it shouldn't affect the size of the panel.  The user might not expect the molecule to move when asked how big it is.  The test drawing is the same if it is removed, so I don't think it's doing anything useful.

Reply:
That's a good point. I will remove that call.